automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspaceApiSpec.scala

```diff
@@ -143,13 +143,13 @@ class WorkspaceApiSpec extends TestKit(ActorSystem("MySpec")) with AnyFreeSpecLi
               case s if s.endsWith("editor") => binding.getMembers.size() shouldEqual 2
               case s if s.endsWith("genomics.serviceAgent") => binding.getMembers.size() shouldEqual 1
               case s if s.endsWith("lifesciences.serviceAgent") => binding.getMembers.size() shouldEqual 1
+              case s if s.endsWith("logging.logWriter") => binding.getMembers() shouldEqual 1
               case s if s.endsWith("owner") =>
                 binding.getMembers.size() shouldEqual 1
                 binding.getMembers.get(0) should endWith("@terra-kernel-k8s.iam.gserviceaccount.com")
               case s if s.endsWith("pubsub.serviceAgent") => binding.getMembers.size() shouldEqual 1
               case other =>
-                logger.error(s"Extra permission on workspace google project found: ${other}")
-                throw new Exception(s"Extra permission on workspace google project found: ${other}")
```

Comment:
It seems like this line may still be important, if we want to be certain about which permissions a project should have, or shouldn't have for auth'z reasons?

Reply:
I'm questioning whether this test is worth keeping. The match statement is still quite brittle even without this exception. An external service adding a new SA or Sam policy group with an existing role (ie increasing the size of one of the role bindings above) will break the test.

Reply:
@marctalbott that's a fair point. Perhaps we just limit to asserting the set diff is sane (i.e,. the "core" set of role bindings are present) and don't even bother with the case match?

Reply:
ok, I think I am sold on removing the case match given it's fragility and proven ability to break the build. I think a check of this nature is still valuable, I will drop a ticket to remediate.

Reply:
@ungwudik does this work for you?

Reply:
Yes, that works.